### ISSUE TITLE
This repository does not comply with the Unix philosophy

### ISSUE THREAD

**Issue by Ivan-Johnson** [Ivan-Johnson/LifeLogServer#10]
There are quite a few non-critical things currently in the repository that add unnecessary complexity. This is evident in the fact that the README file has to have an [entire section](https://github.com/Ivan-Johnson/LifeLog/tree/4fd7634be71a71103504c0934797de1afd4b7b3f#technical-overview-of-the-server) dedicated to explaining what is and is not in the repository. The worst offense is how the repository contains complicated scripts and configuration files that go through the excruciating process of setting up an `nginx` server for forwarding https traffic to LifeLog. Why is this included, while firewall configuration is explicitly excluded? Moreover, why is this included despite the fact that the sort of person who actually wants to install an instance of LifeLog on their server likely already has `nginx` or equivalent setup already.

In short, this repository should contain a minimal set of scripts and code necessary for running a server and basically everything that has anything to do with the following should be entirely discarded or migrated to a new repository:

* nginx

* certbot

* client side code
